lib/pat.c

```diff
@@ -1238,6 +1350,26 @@ pat_node_get_key(grn_ctx *ctx, grn_pat *pat, pat_node *n)
   }
 }
 
+static inline uint8_t *
+_pat_node_get_key(grn_ctx *ctx, grn_pat *pat, pat_node_common *node)
```

Comment:
This function that does the same `pat_node_get_key()`.
If we change `pat_node_get_key()`, we modify many functions.
Because `pat_node_get_key()` uses in many place.
So, I implement function instead of `pat_node_get_key()` temporary.
I will change to `pat_node_get_key()` from `_pat_node_get_key()` in the near feature.